vnext/include/CppWinRTIncludes.h

```diff
@@ -4,22 +4,43 @@
 
 #pragma once
 
+#include <winrt/base.h>
+
 #include <winrt/Windows.Foundation.h>
-#include <winrt/Windows.UI.Composition.h>
-#include <winrt/Windows.UI.Text.h>
-#include <winrt/Windows.UI.Xaml.Controls.Primitives.h>
+
+#include <winrt/Windows.UI.Xaml.h>
+
 #include <winrt/Windows.UI.Xaml.Controls.h>
+
+#include <winrt/Windows.UI.Xaml.Controls.Primitives.h>
+
+#include <winrt/Windows.UI.Xaml.Input.h>
+
+#include <winrt/Windows.UI.Text.h>
+#include <winrt/Windows.UI.Composition.h>
+
 #include <winrt/Windows.UI.Xaml.Hosting.h>
 #include <winrt/Windows.UI.Xaml.h>
```

Comment:
Alphabetize.

Reply:
I rollbacked the change.

---
In reply to: [301440408](https://github.com/microsoft/react-native-windows/pull/2746#discussion_r301440408) [](ancestors = 301440408)